docs/tutorial/part-one/index.md

```diff
@@ -1,390 +1,282 @@
 
-Sit down, buckle up, and let's get going!
+`gatsby new hello-world https://github.com/gatsbyjs/gatsby-starter-hello-world`
 
-## Check your development environment
+When creating a new Gatsby site, you can use the following command structure to create a new site based on any existing Gatsby starter:
 
-Let's check first that you have everything set up to start creating with Gatsby.
-You will need a recent version of Node.js installed.
+`gatsby new [SITE_DIRECTORY] [URL_OF_STARTER_GITHUB_REPO]`
 
-Node.js is a programming tool for running JavaScript on servers and in your
-computer's terminal. Gatsby is built using Node.js.
+> 💡 See a list of the existing [official and community starters](/docs/gatsby-starters/)!
 
-Open a terminal window. See
-[terminal instructions for Mac users](http://www.macworld.co.uk/feature/mac-software/how-use-terminal-on-mac-3608274/) and
-[terminal instructions for Windows users](https://www.quora.com/How-do-I-open-terminal-in-windows). In your terminal window, type `node --version` and hit ENTER, then `npm --version` and hit ENTER (tip: to run a specified command, you must type the command into your terminal and then press ENTER. Then the command will run).
+If you omit a URL from the end, Gatsby will automatically generate a site for you based on the [default starter](https://github.com/gatsbyjs/gatsby-starter-default). Let’s do that now, to create a new site with the default starter, and explore the output.
 
-You should see something like:
+### ✋ Generate a new site based on Gatsby’s default starter
 
-![Check if node.js/npm is installed](check-versions.png)
+1. Open up your terminal. 
+2. Run `gatsby new my-tutorial-site`.
+3. Run `cd my-tutorial-site`.
+4. Run `gatsby develop`.
 
-Gatsby supports versions of Node back to v6 and npm to v3.
+This process should look familiar — it’s almost exactly what we did in the last section to scaffold out the "hello world" site. But this time, because we’re using a different starter, when you visit [http://localhost:8000](http://localhost:8000/) things will look a little different:
 
-If you don't have Node.js installed, go to https://nodejs.org/ and install the
-recommended version for your operating system.
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_E618395E4841D470F48D0AACF52A3EBAEA74345E0FB4AC285CBC937727755E20_1520719644125_Screen+Shot+2018-03-07+at+11.00.28+PM.png)
 
-## Install the "Hello World" starter
 
-Gatsby uses "starters" for starting new projects. Starters
-are partially built Gatsby sites that are pre-configured to help you get moving faster.
-There are several official starters and many others contributed from the Gatsby
-community! [See the Starters page for the full list](/docs/gatsby-starters/).
+Let’s take a look at the code we’ve just generated.
 
-To install a starter, first install Gatsby's command line program by running the following:
+## Gatsby’s project structure
+### ✋ Open up the code using your editor of choice.
```

Comment:
Ah, yes. Another thing we ought to do in part zero is say "install prettier and VS code. You are welcome to opt out of those and use whichever code editor you prefer; this tutorial will assume you are using those." We just want to make sure people know that we aren't maintaining docs that accommodate any possible coding environment.

Reply:
Also, we need to give instructions that they will need to open the entire directory 'my-tutorial-site' in the code editor.